src/ansible_compat/runtime.py

```diff
@@ -461,6 +466,22 @@ def _install_galaxy_role(
             link_path,
         )
 
+    def _update_env(self, varname: str, value: List[str], default: str = "") -> None:
+        """Update colon based environment variable if needed.
+
+        New values are added by inserting them to assure they take precedence
+        over the existing ones.
+        """
+        if value:
+            orig_value = self.environ.get(varname, default)
+            if orig_value:
+                # Prepend original or default variable content to custom content.
```

Comment:
This comment should be removed since it is wrong and the functions documentation already said what are we going to do.